plugins/bat

```diff
@@ -0,0 +1 @@
+repository = https://gitlab.com/wt0f/asdf-bat.git
```

Comment:
Insert a final newline

Reply:
ARG... VisualStudio Code does this to me all the time... It has been fixed. Thanks.